packages/flutter/test/material/data_table_test.dart

```diff
@@ -210,4 +210,59 @@ void main() {
     expect(tester.renderObject<RenderBox>(find.byType(Row).first).size.width, lessThan(800.0));
     expect(tester.takeException(), isNull);
   });
+
+  testWidgets('DataTable column onSort test', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Material(
+          child: DataTable(
+            columns: const <DataColumn>[
+              DataColumn(
+                label: Text('Dessert'),
+              ),
+            ],
+            rows: const <DataRow>[
+              DataRow(
+                cells: <DataCell>[
+                  DataCell(
+                    Text('Lollipop'), // wraps
+                  ),
+                ],
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+    await tester.tap(find.text('Dessert').first);
+    expect(tester.takeException(), isNull);
```

Comment:
I think you have to tester.pump here to make it draw a frame

Reply:
```
testWidgets('DataTable column onSort test', (WidgetTester tester) async {
    await tester.pumpWidget(
      MaterialApp(
        home: Material(
          child: DataTable(
            columns: const <DataColumn>[
              DataColumn(
                label: Text('Dessert'),
              ),
            ],
            rows: const <DataRow>[
              DataRow(
                cells: <DataCell>[
                  DataCell(
                    Text('Lollipop'), // wraps
                  ),
                ],
              ),
            ],
          ),
        ),
      ),
    );
    await tester.tap(find.text('Dessert'));
```
> testWidgets('DataTable column onSort test', (WidgetTester tester) async {
    await tester.pumpWidget(

I think I've called tester.pumpWidget here.

Reply:
It's not a big deal since the original bug was on the onTap but to be more strict, you'd ideally also draw a frame after tapping the column to let it draw the effect of sorting.

Reply:
Got it.